PWGCF/FemtoWorld/DataModel/FemtoWorldDerived.h

```diff
@@ -0,0 +1,249 @@
+                  pidtof_tiny::TOFNSigmaPi<pidtof_tiny::TOFNSigmaStorePi>,
+                  pidtof_tiny::TOFNSigmaKa<pidtof_tiny::TOFNSigmaStoreKa>,
+                  pidtof_tiny::TOFNSigmaPr<pidtof_tiny::TOFNSigmaStorePr>,
+                  pidtof_tiny::TOFNSigmaDe<pidtof_tiny::TOFNSigmaStoreDe>);
+using FemtoWorldParticle = FemtoWorldParticles::iterator;
+/*
+DECLARE_SOA_TABLE(FemtoWorldDebugParticles, "AOD", "FEMTOWORLDDEBUGPARTS",
+                  femtoworldparticle::Sign,
+                  femtoworldparticle::TPCNClsFound,
+                  track::TPCNClsFindable,
+                  femtoworldparticle::TPCNClsCrossedRows,
+                  track::TPCNClsShared,
+                  track::TPCInnerParam,
+                  femtoworldparticle::ITSNCls,
+                  femtoworldparticle::ITSNClsInnerBarrel,
+                  track::DcaXY,
+                  track::DcaZ,
+                  track::TPCSignal,
+                  pidtpc_tiny::TPCNSigmaStoreEl,
+                  pidtpc_tiny::TPCNSigmaStorePi,
+                  pidtpc_tiny::TPCNSigmaStoreKa,
+                  pidtpc_tiny::TPCNSigmaStorePr,
+                  pidtpc_tiny::TPCNSigmaStoreDe,
+                  pidtof_tiny::TOFNSigmaStoreEl,
+                  pidtof_tiny::TOFNSigmaStorePi,
+                  pidtof_tiny::TOFNSigmaStoreKa,
+                  pidtof_tiny::TOFNSigmaStorePr,
+                  pidtof_tiny::TOFNSigmaStoreDe,
+                  femtoworldparticle::DaughDCA,
+                  femtoworldparticle::TransRadius,
+                  femtoworldparticle::DecayVtxX,
+                  femtoworldparticle::DecayVtxY,
+                  femtoworldparticle::DecayVtxZ,
+                  femtoworldparticle::MKaon,
+                  femtoworldparticle::TPCCrossedRowsOverFindableCls<track::TPCNClsFindable, femtoworldparticle::TPCNClsCrossedRows>,
+                  pidtpc_tiny::TPCNSigmaEl<pidtpc_tiny::TPCNSigmaStoreEl>,
+                  pidtpc_tiny::TPCNSigmaPi<pidtpc_tiny::TPCNSigmaStorePi>,
+                  pidtpc_tiny::TPCNSigmaKa<pidtpc_tiny::TPCNSigmaStoreKa>,
+                  pidtpc_tiny::TPCNSigmaPr<pidtpc_tiny::TPCNSigmaStorePr>,
+                  pidtpc_tiny::TPCNSigmaDe<pidtpc_tiny::TPCNSigmaStoreDe>,
+                  pidtof_tiny::TOFNSigmaEl<pidtof_tiny::TOFNSigmaStoreEl>,
+                  pidtof_tiny::TOFNSigmaPi<pidtof_tiny::TOFNSigmaStorePi>,
+                  pidtof_tiny::TOFNSigmaKa<pidtof_tiny::TOFNSigmaStoreKa>,
+                  pidtof_tiny::TOFNSigmaPr<pidtof_tiny::TOFNSigmaStorePr>,
+                  pidtof_tiny::TOFNSigmaDe<pidtof_tiny::TOFNSigmaStoreDe>);
+using FemtoWorldDebugParticle = FemtoWorldDebugParticles::iterator;
+*/
+/// Hash
+/*
+namespace hash
+{
+DECLARE_SOA_COLUMN(Bin, bin, int); //! Hash for the event mixing
+} // namespace hash
+DECLARE_SOA_TABLE(Hashes, "AOD", "HASH", hash::Bin);
+using Hash = Hashes::iterator;
+
+} // namespace o2::aod
+*/
```

Comment:
Please get rid of these comments
Data models should be clean to not infer to wrong interpretations
You always have the original in femto dream in case you need it